packages/kbn-unified-data-table/src/components/column_header_truncate_container.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiTextBlockTruncate } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+const ColumnHeaderTruncateContainer = ({
+  headerRowHeight = 1,
+  children,
+}: {
+  headerRowHeight: number;
+  children: React.ReactNode;
+}) => {
+  return (
+    <EuiTextBlockTruncate
+      lines={headerRowHeight}
+      css={css`
+        overflow-wrap: anywhere;
+        white-space: normal;
+        word-break: break-all;
```

Comment:
**doubt**: not sure what is the expected styling here, do you want to perform word breaking on long words?
If that's the case, the [`eui-textBreakAll`](https://eui.elastic.co/#/theming/typography/utilities#wrapping) utility class might be useful here.

Reply:
I believe that's the case, but i don't have further insights on this code. I simply moved it to another file for reusability. @mbondyra do you know ?